### PR TITLE
Milestone 4: Add contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by yemidada**
In this milestone, I
- completed the design by included the contact form
- Utilize Flexbox and Grid to position various elements on a web page.
- Develop a portfolio website.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,226 @@
...
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
```
Comment (usorfaitheloho) [inline README.md]: Please remove all commented lines from your README file to ensure you have a clean code free from confusion and easy to maintain 😄

File: README.md [hunk 2]
```diff
@@ -1 +1,226 @@
...
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
```
Comment (usorfaitheloho) [inline README.md]: Please remove all commented lines from your README file to ensure you have a clean code from confusion and easy to maintain 😄

File: README.md [hunk 3]
```diff
@@ -1 +1,226 @@
...
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <h3><b>Portfolio: add a contact form and finish the mobile version</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
```
Comment (usorfaitheloho) [inline README.md]: Please remove all commented lines from your README file to ensure you have a clean code from confusion and easy to maintain 😄

File: README.md [hunk 4]
```diff
@@ -1 +1,226 @@
...
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
```
Comment (usorfaitheloho) [inline README.md]: Please remove all commented lines from your README file to ensure you have a clean code from confusion and easy to maintain 😄

File: README.md [hunk 5]
```diff
@@ -1 +1,226 @@
...
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
```
Comment (usorfaitheloho) [inline README.md]: Please remove all commented lines from your README file to ensure you have a clean code from confusion and easy to maintain 😄

File: README.md [hunk 6]
```diff
@@ -1 +1,226 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- AUTHORS -->
```
Comment (usorfaitheloho) [inline README.md]: Please remove all commented lines from your README file to ensure you have a clean code from confusion and easy to maintain 😄

File: README.md [hunk 7]
```diff
@@ -1 +1,226 @@
...
+👤 **Author**
+
+- GitHub: [@yemidada](https://github.com/yemidada)
+- Twitter: [@naijavine](https://twitter.com/naijavine)
+- LinkedIn: [yemidada](https://linkedin.com/in/yemidada)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
```
Comment (usorfaitheloho) [inline README.md]: Please remove all commented lines from your README file to ensure you have a clean code from confusion and easy to maintain 😄

File: README.md [hunk 8]
```diff
@@ -1 +1,226 @@
...
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```
Comment (usorfaitheloho) [inline README.md]: Please remove all descriptions from your README file to ensure you have a clean code from confusion and easy to maintain 😄

File: README.md [hunk 9]
```diff
@@ -1 +1,226 @@
...
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **Add animation effect to the css (Animate.css)**
+- [ ] **Improve the designs**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
```
Comment (usorfaitheloho) [inline README.md]: Please remove all commented lines from your README file to ensure you have a clean code from confusion and easy to maintain 😄

File: README.md [hunk 10]
```diff
@@ -1 +1,226 @@
...
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
```
Comment (usorfaitheloho) [inline README.md:142]: Please remove all commented lines from your README file to ensure you have a clean code from confusion and easy to maintain 😄

File: README.md [hunk 11]
```diff
@@ -1 +1,226 @@
...
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can support us  by sharing our code
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
```
Comment (usorfaitheloho) [inline README.md]: Please remove all commented lines from your README file to ensure you have a clean code from confusion and easy to maintain 😄

File: README.md [hunk 12]
```diff
@@ -1 +1,226 @@
...
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Thanks you for checking out my repository
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
```
Comment (usorfaitheloho) [inline README.md]: Please remove all commented lines from your README file to ensure you have a clean code from confusion and easy to maintain 😄

File: README.md [hunk 13]
```diff
@@ -1 +1,226 @@
...
+
+  - Yes, nodejs need to be running properly on your machine
+
+- **Can you test linter on your local machine and github**
+
+  - Yes, you can do both
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
```
Comment (usorfaitheloho) [inline README.md]: Please remove all commented lines from your README file to ensure you have a clean code from confusion and easy to maintain 😄

File: index.html [hunk 14]
```diff
@@ -0,0 +1,354 @@
...
+          required
+          placeholder="loremipsum@gmail.com"
+          name="name"
+          type="email"
+        />
+        <textarea
+          rows="7"
+          placeholder="Write your message here..."
+          maxlength="500"
+        ></textarea>
```
Comment (usorfaitheloho) [inline index.html]: You did well adding all the validations for the other input elements, please do well adding the `required` element for the `textarea` as well to enable a user fill all the input in the form before it is submitted 😄

File: assets/css/style.css [hunk 15]
```diff
@@ -0,0 +1,646 @@
...
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  input,
+  textarea {
+    width: 447px;
+  }
+}
```
Comment (usorfaitheloho) [inline assets/css/style.css]: Your contact form section is missing all the background images in the original design, please do well to include them in yours to enable it to have a good aesthetic for visitors and hiring managers 😄 

Figma design :

image

Your design :

image

File: assets/css/style.css [hunk 16]
```diff
@@ -0,0 +1,646 @@
...
+  text-align: center;
+  color: white;
+  margin-bottom: 12px;
+}
+
+.footer .desc {
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: rgba(235, 235, 255, 1);
```
Comment (usorfaitheloho) [inline assets/css/style.css]: Please use  the right color given in the Figma template to have good design that matches with the original one 😄

File: assets/css/style.css [hunk 17]
```diff
@@ -0,0 +1,646 @@
...
+input,
+textarea {
+  padding: 15px;
+  margin: 8px;
+  border: none;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+}
```
Comment (usorfaitheloho) [inline assets/css/style.css]: Please use  the right color for all the placeholders given in the Figma template to have good design that matches with the original one 😄

File: assets/css/style.css [hunk 18]
```diff
@@ -0,0 +1,646 @@
...
+
+.footer {
+  flex-direction: column;
+  position: relative;
+  border-top-left-radius: 5rem;
+  z-index: 10;
+  background-color: rgba(96, 112, 255, 1);
+  justify-content: center;
+  align-items: center;
+}
```
Comment (usorfaitheloho) [inline assets/css/style.css]: Please I'd suggest you add the required spacing and margin needed as it is in the design, to enable your portfolio to have a better UI design 😄 : 

Your design :

image

figma design :
image

File: assets/css/style.css [hunk 19]
```diff
@@ -0,0 +1,646 @@
...
+    flex-direction: row;
+    width: 100%;
+    padding-left: 0.5rem;
+  }
+
+  .footer .button-outline {
+    justify-content: flex-start;
+    flex-flow: column;
+  }
+
```
Comment (usorfaitheloho) [inline assets/css/style.css:519]: Please add the button interaction styling with the right colors for your design as it is stated in the figma template 😄 

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @yemidada  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!  [ 68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966 ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966) [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Added the contact form using the Figma design given ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 20]
```diff
@@ -1 +1,177 @@
...
+- **Can you test linter on your local machine and github**
+
+  - Yes, you can do both
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+ This project is [MIT](./MIT.md) licensed.
```
Comment (Olamarx) [inline README.md:174]: - [OPTIONAL] Kindly resolve this url, it goes to the 404 page.

**Review by Olamarx — APPROVED** [review]
Hi @yemidada, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️
- Readme file professional ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 20 above)
Comment (yemidada) [reply README.md:174]: Thank you very much. I have fixed the broken link @Olamarx

**Pull request merged**